ui/src/App.tsx

```diff
@@ -330,36 +328,9 @@ const AppComponent = () => {
         path: paths.app.notifications,
         render: () => <Notifications notifications={notifications} />,
       },
-    ],
-  });
-
-  entries.push({
-    displayEntry: () => tradingService.length > 0,
-    sidebar: [
-      {
-        label: 'Markets',
-        path: paths.app.markets.root,
-        activeSubroutes: true,
-        render: () => <Markets listings={listings} />,
-        icon: <OrdersIcon />,
-        groupBy: 'Secondary Market',
-        children: listings.map(c => ({
-          label: c.payload.listingId.label,
-          path: paths.app.markets.root + '/' + c.contractId.replace('#', '_'),
-          render: () => <Market services={tradingService} cid={c.contractId} listings={listings} />,
-          icon: <ExchangeIcon />,
-          children: [],
-        })),
-      },
-    ],
-    additionalRoutes: [
-      {
-        path: paths.app.markets.order + '/:contractId',
-        render: () => <TradingOrder listings={listings} />,
-      },
       {
-        path: paths.app.markets.offer,
-        render: () => <Offer service={ServiceKind.TRADING} />,
+        path: paths.app.setup.identity,
```

Comment:
If we're removing the setup page, does it still make sense to have a `/app/setup/identity` path?

Reply:
yeah good point ill remove that

Reply:
bye bye setup